### PR TITLE
feat: add remaining fields to confirm sign in

### PR DESCRIPTION
**Pull request by Jordan-Nelson**
*Issue #, if available:*

*Description of changes:*
- add remaining fields to confirm sign in

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_authenticator/lib/src/widgets/form_fields/sign_up_form_field.dart [hunk 1]
```diff
@@ -251,7 +251,7 @@ abstract class SignUpFormField<FieldValue> extends AuthenticatorFormField<
     Key? key,
     required String title,
     required CognitoUserAttributeKey attributeKey,
-    required String hintText,
```
Comment (Jordan-Nelson) [inline packages/amplify_authenticator/lib/src/widgets/form_fields/sign_up_form_field.dart:254]: I updated this for consistency with Confirm Sign In. I don't think hint text needs to be required. It isn't required for TextFormField / InputDecoration, and it makes sense to try to match to APIs.

**Review by dnys1 — APPROVED** [review]
Looks good - thanks!

File: packages/amplify_authenticator/lib/src/widgets/form_fields/sign_up_form_field.dart (hunk 1 above)
Comment (haverchuck) [reply packages/amplify_authenticator/lib/src/widgets/form_fields/sign_up_form_field.dart:254]: What happens if the customer creates a custom form field and doesn't supply hint text?

Comment (Jordan-Nelson) [reply packages/amplify_authenticator/lib/src/widgets/form_fields/sign_up_form_field.dart:254]: It just won't display any hint text then

Screen Shot 2021-11-23 at 12 51 31 PM
y

**Review by haverchuck — APPROVED** [review]

**Pull request merged**